src/js/overlay.js

```diff
@@ -201,8 +201,6 @@ Overlay.prototype.show = function() {
 		document.body.appendChild(shadow);
 	}
 
-	this.content.focus();
```

Comment:
I need to look into improving accessibility, but having that doesn't seem to change anything, that's why I removed it